.eslintrc.js

```diff
@@ -1,21 +1,31 @@
 module.exports = {
     root: true,
-
-    extends: 'standard',
-
-    env: {
-        node: false // Opt-in per file to node env, globals, etc.
-    },
-
+    parser: '@typescript-eslint/parser',
+    plugins: ['@typescript-eslint'],
+    extends: [
+        'eslint:recommended',
+        'plugin:@typescript-eslint/recommended'
+    ],
     rules: {
-        semi: ['error', 'always'],
+        'semi': ['error', 'always'],
         'no-extra-semi': 'error',
-        indent: ['error', 4, { 'SwitchCase': 1 }],
+        'no-unused-vars': 'off',
+        'no-undef': 'off',
+        'no-prototype-builtins': 'off',
+        'indent': ['error', 4, { 'SwitchCase': 1 }],
         'space-before-function-paren': ['error',
             {
                 'anonymous': 'always',
                 'named': 'never'
             }
-        ]
+        ],
+        '@typescript-eslint/ban-types': 'off',
+        '@typescript-eslint/explicit-module-boundary-types': 'off',
+        '@typescript-eslint/no-var-requires': 'off',
+        // TODO: turn these on when project is split into separate src/test folders
```

Comment:
So one thing we should talk about a bit @justinaray is potentially splitting the test files out into their own folder that way we can have separate tsconfig files for each. This would give us the same behavior as before with the environment opt-in for node/jasmine, but with proper types. We could also have separate eslintrc files which could turn some of these more restrictive rules, that we need off for testing, on for source files.

Reply:
@jakejrichards ... yeah, completely agree.  let's move them over to a tests folder when this one lands.